uqcsbot/scripts/wiki.py

```diff
@@ -0,0 +1,26 @@
+from uqcsbot import bot, Command
+import requests
+import json
+
+
+@bot.on_command("wiki")
+async def handle_wiki(command: Command):
+    search = command.arg
+    url = f"https://en.wikipedia.org/w/api.php?action=opensearch&search={search}&format=json"
```

Comment:
using fstrings here seems awkward-- I think it'd be better to pass 'search' as a kwarg like here: http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls, just inline is fine